builder/amazon/ebs/builder.go

```diff
@@ -173,6 +174,9 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 				b.config.RunConfig.Comm.SSHPassword),
 		},
 		&common.StepProvision{},
+	}
+
+	amiCreationSteps := []multistep.Step{
 		&awscommon.StepStopEBSBackedInstance{
```

Comment:
This is taken care of by instance termination in the cleanup phase for StepRunSourceInstance:
https://github.com/jeremy-asher/packer/blob/59109d6786f58b18e89362cc7b999c2361469735/builder/amazon/common/step_run_source_instance.go#L347

https://github.com/jeremy-asher/packer/blob/59109d6786f58b18e89362cc7b999c2361469735/builder/amazon/ebs/builder.go#L138